### ISSUE TITLE
Gamma setting looks like it can be changed but can't

### ISSUE THREAD

**Issue by copygirl** [anegostudios/VintageStory-Issues#266]
If the setting is supposed to be disabled it should probably look grayed out. But really, it should be modifiable. I would argue that it's an essential accessibility option.

**Comment by tyronx** [comment]
Ok fixed for now.
As mentioned before, for a modifiable thing I want it to have its own screen where you are encouraged to set the right gamma according to a reference image.
